BLSQ/iaso
# Polio-1149: vaccine stock list view(mock data)

Table + Filters for stock management list view

Related JIRA tickets : POLIO-1149

## Self proofreading checklist


## Changes

- Add menu entry, hidden behind `SHOW_DEV_FEATURES` feature flag
- Add filters  deep linking
- Add table
- Add api hook using mock data
- mock table data

## How to test
- Enable `SHOW_DEV_FEATURES` feature flag on your local account
- Go to polio>vaccine module>stock manbagement
- Play around with the table and filters: displayed results won't change but the url should update correctly
- Test deep linking

## Print screen / video

https://github.com/BLSQ/iaso/assets/38907762/c2d1e584-cb18-4641-9393-602847418ef1



## Notes

This PR is using mock data. There's a separate ticket to plug the front-end into the backend. 
Since the feature is hidden behind a feature flag, clients will not see it even if it's deployed.
